gumstix_raspberry_pi_zero_battery_board_demo/README.md

```diff
@@ -0,0 +1,115 @@
+# Gumstix Raspberry Pi Zero Battery Board Demo
+
+## Download and flash Gumstix’s Raspbian Lite image 
+Download the Gumstix Raspbian Lite image from:
+https://gumstix-raspbian.s3.amazonaws.com/2020-01-12/raspberrypi-cm3/rpi-4.19.y/2019-09-26-raspbian-buster-lite.img.xz
+
+Flash your SD card with the extracted image above. To do this, open a terminal on your host computer. 
+```
+$ sudo dd if=2019-09-26-raspbian-buster-lite.img of=/dev/sdX bs=4MiB
+```
+Where sdX is the location of your SD card (to find the location of the SD card, you can use the command “lsblk” to match the memory device with the SD card). 
+
+## Setting up SSH:
+
+In the boot partition, create an empty file called “ssh” 
+
+In the boot partition, create a file called wpa_supplicant.conf. Enter your WIFI_SSID and WIFI_PASSWORD for the ssid and psk respectively. You will also need to fill in the entry for “country” with the ISO code of your country (eg: Canada = CA).
+Within it, copy and paste the following lines:
+```
+country=ISO_code_of_your_country
+ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+update_config=1
+
+network={
+ssid="WIFI_SSID"
+scan_ssid=1
+psk="WIFI_PASSWORD"
+key_mgmt=WPA-PSK
+}
+```
+
+In the rootfs, navigate to /etc/ssh/sshd_config. Edit the sshd_config file, by adding the following line to the end of the file: 
+```
+IPQoS cs0 cs0
+```
+If you have trouble saving the sshd_config file, try running “sudo su”, and then try editing and saving the sshd_config file.
+
+Insert the SD card into your Raspberry Pi Zero. Now your Raspberry Pi Zero should be accessible via SSH 
+
+Power up your Raspberry Pi Zero and give it a minute to boot up. Then, in a terminal on the host computer, enter the following command:
+```
+$ ssh pi@raspberrypi.local
+```
+Now you should have SSH access to the Raspberry Pi Zero.
+
+## Enable the Camera
+```
+$ sudo raspi-config
+```
+
+Navigate to “Interfacing Options”, and enable the Camera. The Raspberry Pi Zero will ask you if you want to reboot after this, select “Yes”. It’ll end the ssh session, and you’ll have to again ssh into the Raspberry Pi Zero by running: “ssh pi@raspberrypi.local” in a terminal. 
+
+## Update your Raspberry Pi Zero
+```
+$ sudo apt update
+```
+
+## Install Gstreamer1.0
+```
+$ sudo apt-get install gstreamer1.0-tools
+$ sudo apt-get install gstreamer1.0-plugins-bad
+$ sudo apt-get install gstreamer1.0-plugins-good
+$ sudo apt-get install gstreamer1.0-omx
+$ sudo apt-get install python3-gi
+$ sudo apt-get install python-gst-1.0
```

Comment:
Suggestion: This can be abreviated to a single command:
```
$ sudo apt-get install gstreamer1.0-tools \
              gstreamer1.0-plugins-bad \
              gstreamer1.0-plugins-good \
              gstreamer1.0-omx \
              python3-gi \
              python-gst-1.0 
```

Reply:
Will implement this.